src/Bundle/Form/DataTransformer/ResourceToIdentifierTransformer.php

```diff
@@ -33,13 +33,9 @@ public function __construct(RepositoryInterface $repository, ?string $identifier
     }
 
     /**
-     * @psalm-suppress MissingParamType
-     *
-     * @param object|null $value
-     *
-     * @return mixed
+     * @inheritDoc
      */
-    public function transform($value)
+    public function transform($value): mixed
```

Comment:
```suggestion
    public function transform(mixed $value): mixed
```
https://github.com/symfony/symfony/blob/bed9c3660c0106049622eead422fcb3921f8dd21/src/Symfony/Component/Form/DataTransformerInterface.php#L65

Reply:
```php
    /** @param int|string|null $value */
public function reverseTransform($value): ?ResourceInterface
```
Not sure how it's even working, since correct typehint is 
```php
 public function reverseTransform(mixed $value): mixed;
```

Reply:
you mean for the return typehint? this is ok, you can restrict it but I don't even remember on which php version.